app/src/main/java/com/example/triptracker/view/profile/UserProfileOverview.kt

```diff
@@ -67,199 +72,264 @@ fun UserProfilePreview() {
+                            Text(
+                                text = "${myTripsCount}", //Call to the filtered Itinerarylist
+                                modifier = Modifier.align(Alignment.CenterHorizontally),
+                                style = AppTypography.bigNumberStyle
+                            )
+                            Text(
+                                text = "Trips",
+                                modifier = Modifier.align(Alignment.CenterHorizontally),
+                                style = AppTypography.categoryTextStyle
+                            )
+                        }
+                        Column(
+                            modifier = Modifier.align(Alignment.CenterVertically)
+                                .padding(horizontal = 30.dp)
+                        ) {
+                            Text(
+                                text = "${profile.followers.size}",
+                                modifier = Modifier.align(Alignment.CenterHorizontally),
+                                style = AppTypography.bigNumberStyle
+                            )
+                            Text(
+                                text = "Followers",
+                                modifier = Modifier.align(Alignment.CenterHorizontally),
+                                style = AppTypography.categoryTextStyle
+                            )
+                        }
+                        Column(
+                            modifier = Modifier.align(Alignment.CenterVertically)
+                                .padding(horizontal = 30.dp)
+                        ) {
+                            Text(
+                                text = "${profile.following.size}",
+                                modifier = Modifier.align(Alignment.CenterHorizontally),
+                                style = AppTypography.bigNumberStyle
+                            )
+                            Text(
+                                text = "Following",
+                                modifier = Modifier.align(Alignment.CenterHorizontally),
+                                style = AppTypography.categoryTextStyle
+                            )
+                        }
+                    }
+                    // Favourites, Friends, Settings and MyTrips tiles
+                    Box(
+                        modifier =
+                        Modifier.height(300.dp).width(350.dp).align(Alignment.CenterHorizontally)
+                    ) {
+                        ProfileButton(
+                            label = "Favourites",
+                            icon = Icons.Outlined.FavoriteBorder,
+                            onClick = {
+                                navigation.navigateTo(
+                                    TopLevelDestination(
+                                        Route.FAVORITES, Icons.Outlined.Favorite, "Favorites"
+                                    )

```

Comment:
Maybe we can use the navigation.navcontroller.navigate(ROUTE) function since these top level destinations are not real top level destinations ? 
Or if they are added to the top level then they should be put in the Navigation.kt